requirements/static/pkg/py3.6/linux.txt

```diff
@@ -45,6 +45,8 @@ jinja2==3.0.3
     # via -r requirements/base.txt
 jmespath==0.10.0
     # via -r requirements/base.txt
+looseversion==1.0.2
```

Comment:
Maybe I'm missing something, but these requirements are stating they are from requirements/base.txt, but I'm not seeing that file being edited.

Reply:
It was removed from requirements/base.txt, that is, looseversion and packaging, etc. were removed.
Perhaps the Py3.6 static requirements were not regenerated (that is removed) since the Py 3.6 support is comment out in pre-commit.

Could remove all of the Py3.6 from the requirements/static/[ci|pkg] directories, if you feel appropriate, but wonder about the old tests running ?